data/engines.txt

```diff
@@ -419,7 +419,7 @@ outfit "A860 Atomic Thruster"
 	"flare sprite" "effect/atomic flare/huge"
 		"frame rate" 10
 	"flare sound" "atomic huge"
-	description "The A860 atomic thruster is the most powerful engine sold in human space, and is much more compact than any of the alternatives. But, you may need to upgrade your ship's power and cooling systems to handle it."
+	description "The A860 atomic thruster is the most powerful engine sold in human space, and is much more compact than any of the alternatives. However, you may need to upgrade your ship's power and cooling systems to handle it."
```

Comment:
Something like this might work (though it's quite wordy):

> While this beast might require an upgrade to power and cooling systems before your ship can handle it, there is no doubt that the immense power of the thrusters is well worth it for any ship with enough engine space. Not even the new dent in your wallet will seem significant next to the outstanding performance of this engine.